server/GQLFactory/helpers/helperFunctions.js

```diff
@@ -37,6 +37,7 @@ const typeConversion = {
   date: 'String',
   boolean: 'Boolean',
   numeric: 'Int',
+  bigint: 'String',
```

Comment:
We just want to clarify why bigint will return a string -- isn't this an integer?

Reply:
We don't think int can handle the size of bigint numbers so we store them as strings instead, in case of error.